### DISCUSSION TITLE
Autodesk Flame support

### DISCUSSION THREAD

**Discussion by finnschi** [videovillage/community#199] [Ideas]
Flame supports OFX and the Filmbox plugin seems to load and the UI elements show up for me to dial in stuff, but thats about it.

We would love to somehow transfer stuff like grain and halation nodes to render them in Flame , we can allready render native resolve grades using their OFX plugin.  

We often do format adaptions (1x1, 9x16) and such in flame , having timeline-resolution grain is better than dealing with per clip grain. 

So we would absolutely LOVE flame support, even if its just a reader like BLG is, but as its OFX maybe its easy, maybe the baselight version allready works in Flame? I dont know

**Comment by gregcotten** [comment]
Once they support OpenFX 1.5 + Metal and CUDA rendering paths it should work without issue.

**Comment by dr1pvfx** [comment]
FWIW in flame2026 filmbox seems to work, only tried with the LITE version but it does things.

**Comment by BMvac** [comment]
The LITE version "sort of" works in Flame 2026. 
- it is not correctly recognizing Flame OCIO color policy: the correct result requires an inverse color transform after the ofx node, only Rec 709 is correctly utilized/recognized?
- Many setting don't work/no response to adjustment. These are:
Advanced Settings
Lab
Split Tone
Debug

It would be great to be able to use this in Flame. Like mentioned applying this on the timeline level in finishing is the more ideal and visually consistent way.

**Reply by gregcotten** [reply]
Flame's OpenFX implementation is not up to par. If they want OFX plugins to be colorspace-aware they need to implement the OpenFX 1.5 Colour Management extensions. They are also not respecting menu hiding correctly - no Debug menus should ever be exposed to the user, and on every other host they are hidden.

**Reply by BMvac** [reply]
Gotcha, my understanding was they were supporting the most recent OFX version but will check.

**Reply by gregcotten** [reply]
OpenFX has a lot of "opt-in" features on both the plugin and host side. Support for the new OFX 1.5 colour management feature is not strictly required for a host to be 1.5 "compliant"

**Comment by tomothyhondrox** [comment]
I just tried the free trial of the new full version with Flame 2026.1 on Linux, no dice.  The color transforms actually work quite well now, no matter what colorspace I feed the node I get the same output... But there seems to be no change in visuals when I try to choose different film stocks.  A lot of commercials are finished in Flame still, with VFX and conform done after color.  So I think you'd get a lot of customers out of it if you were able to tweak it and make it work.   Of course Autodesk is gonna Autodesk and makes your lives difficult and if you can't swing it I totally understand.
